contentcuration/contentcuration/static/js/edit_channel/channel_set/vuex/getters.js

```diff
@@ -0,0 +1,46 @@
+var _ = require('underscore');
+
+exports.isNewSet = function(state) {
```

Comment:
Depending on how frequently you are using these, you can also define these inline with the `mapState` helper in the components where they get used (i.e. if they only get used in a single component) - https://vuex.vuejs.org/guide/state.html#the-mapstate-helper